packages/terra-action-header/src/terra-dev-site/doc/example/BackActionHeader.jsx

```diff
@@ -0,0 +1,18 @@
+/* eslint-disable no-alert */
+import React from 'react';
+// eslint-disable-next-line import/no-extraneous-dependencies, import/no-unresolved, import/extensions
```

Comment:
Is this lint ignore still applicable?

Reply:
Yes, the next line is resolved by webpack and will fail the linter because it take into account webpack setup.